.github/actions/deploy-ssr-web/action.yml

```diff
@@ -0,0 +1,43 @@
+name: Deploy SSR web
+description: Deploy a FastBoot server serving a stripped down web-client
+
+inputs:
+  waypoint_version:
+    description: Waypoint version to use
+    required: false
+    default: '0.4.0'
+  environment:
```

Comment:
I think this is not needed anymore because we're always building a production build here

Reply:
In the web client deployment, it’s used for the [deployment script](https://github.com/cardstack/cardstack/blob/main/.github/actions/deploy-web-client/action.yml#L19) (which ultimately sets the `DEPLOY_TARGET` for Ember CLI Deploy) and [configuring Sentry](https://github.com/cardstack/cardstack/blob/main/.github/actions/deploy-web-client/action.yml#L41). Maybe we’ll need both of these? Not sure about how Hub URL is configured, will look more into it.